internal/livelint/interaction.go

```diff
@@ -0,0 +1,60 @@
+package livelint
+
+import (
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+//go:generate moq -out mock_ui_test.go -pkg livelint_test . UserInteraction
+
+type UserInteraction interface {
+	DisplayVerboseSetting(verbose bool)
+	DisplayContext(contextMsg string)
+	DisplayCheckStart(checkMsg string)
```

Comment:
Would it make sense to pass a `checkName` or `checkID` to all three of the check functions? It seems that they should be called sequentially and if we send checks in parallel at some point in the future, this would help with disambiguation.

Reply:
Same here. I can see those parameters being useful in the future, but I would add them when we need them.